apps/modernization-ui/src/pages/patient/profile/identification/IdentificationsTable.tsx

```diff
@@ -75,13 +78,44 @@ export const IdentificationsTable = ({ patient }: Props) => {
     const { changed } = useProfileContext();
 
     const [isActions, setIsActions] = useState<any>(null);
-    const [identifications, setIdentifications] = useState<PatientIdentification[]>([]);
+
+    const asTableBody =
+        (index?: number) =>
+        (identification: PatientIdentification): TableBody => ({
+            id: identification.__typename,
+            tableDetails: [
+                {
+                    id: 1,
+                    title: identification?.asOf && format(identification.asOf, 'MM/dd/yyyy')
```

Comment:
suggestion: The `internalizeDate` function will do this formatting and also handles `null` or `undefined` input